configs/peer/genesis.json

```diff
@@ -62,6 +62,162 @@
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_set_key_value_in_user_assets"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_remove_key_value_in_user_assets"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_set_key_value_in_user_metadata"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_remove_key_value_in_user_metadata"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_set_key_value_in_asset_definition"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_remove_key_value_in_asset_definition"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_mint_user_asset_definitions"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_transfer_user_assets"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "Register": {
+            "object": {
+              "Raw": {
+                "Identifiable": {
+                  "PermissionTokenId": {
+                    "name": "can_transfer_only_fixed_number_of_times_per_period"
+                  }
+                }
+              }
+            }
+          }
+        },
```

Comment:
now we have to explicitly register all the permissions in the genesis instead of having them implicitly defined?

Reply:
could these permission registrations be replaced with one role registration? What happens if user registers a role containing permissions that weren't registered previously?

Reply:
This in and of itself isn't a problem, but we need to cut down on the noise. Here we have a long sequence of essentially redundant information which is completely unreadable. 

Our RISC + Vector approach would reduce this down to 

```
Register": {
             "object": {
               "Raw": {
                 "Identifiable": {
                   "PermissionTokenId": {
                     "names": [
                                "can_transfer_only_fixed_number_of_times_per_period", 
                                "can_transfer_user_assets"
                   }
                 }
               }
             }
```

This compression would also reduce the on-disk and in memory size of the permission tokens.

Reply:
@mversic 

> now we have to explicitly register all the permissions in the genesis instead of having them implicitly defined?

Yes, that's mostly the point of this PR.

> could these permission registrations be replaced with one role registration?

No, because...

> What happens if user registers a role containing permissions that weren't registered previously?

...`FindError`